src/type.ts

```diff
@@ -18,13 +18,6 @@ export function isType(t: any): t is Type {
   return hasOwnProperty(Type, t);
 }
 
-export function isContinuous(type: Type): type is 'quantitative' | 'temporal' {
-  return type === 'quantitative' || type === 'temporal';
-}
-export function isDiscrete(type: Type): type is 'ordinal' | 'nominal' {
-  return type === 'ordinal' || type === 'nominal';
-}
-
```

Comment:
I replaced these with isDiscrete from https://github.com/vega/vega-lite/blob/11dea5338accc1ed81a9c9a3fbc41d5c6d82cc05/src/channeldef.ts#L870. The semantics are slightly different in that we also consider whether a channeldef is discretized via bin or geojson. I think this is correct but please check carefully.

Reply:
Is this change in functions required to be able to export types? If not, could we save this adjustment for a separate PR ?